README.md

```diff
@@ -609,6 +622,38 @@ Currently there exist three conversation history cache implementations:
 1. Redis cache
 1. Postgres cache
 
+Entries stored in cache have compound keys that consist of `user_id` and `conversation_id`. It is possible for one user to have multiple conversations and thus multiple `conversation_id` values at the same time. Global cache capacity can be specified. The capacity is measured as number of entries; entries size are ignored in this computation.
+
+#### In-memory cache
+
+In-memory cache is implemented as a queue with defined maximum capacity specified as number of entries that can be stored in a cache. That number is limit for all cache entries, not matter how many users are using LLM. When new entry is put into the cache and if the maximum capacity is reached, oldest entry is removed from the cache.
```

Comment:
as a queue with **a** defined maximum capacity specified as **the** number of entries...

Reply:
That number is **the** limit for all cache entries

Reply:
When **the** new entry (...), **the** oldest entry...

Reply:
fixed, thank you